ci/update-requirements.py

```diff
@@ -50,7 +50,7 @@ def pip_compile(target, *sources, upgrade=False):
 
 def update_openvino_dev_reqs():
     package_downloading_stdout = subprocess.run(
-        [sys.executable, '-m', 'pip', 'download', 'openvino-dev', '--no-deps'],
```

Comment:
no deps was added intentionally, to override changes for upcoming release, please stay this script without changes

Reply:
Reverted.